packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -150,11 +150,11 @@ class InteractiveViewer extends StatefulWidget {
        child = null;
 
   /// When laying out the child, InteractiveViewer applies a transform based on the current
-  /// [Matrix4] defined in the [TransformationController] and will translate all the hit tests as well.
+  /// Matrix4 defined in the TransformationController and will translate all the hit tests as well.
```

Comment:
@rodydavis Is there a reason why you removed the types here? You can keep these if you put a

```
/// @docImport 'package:flutter/widgets.dart';
library;
```
or other required import

Reply:
I was seeing if it was the issue for the analyze part. But once it was green I was going to add then back to see what it was failing on